templates/partners/find-a-partner.html

```diff
@@ -2,254 +2,158 @@
+            <div class="p-side-navigation__drawer-header">
+              <a href="#" class="p-side-navigation__toggle--in-drawer" aria-controls="drawer" id="toggle-filters">
+                Toggle filters
+              </a>
+            </div>
+            <div>
+              <aside class="p-accordion" data-multiple-expanded="true">
+                <ul class="p-accordion__list">
+                  <li class="p-accordion__group" id="programme-section">
+                    <div role="heading" aria-level="2">
+                      <button type="button" class="p-accordion__tab u-no-padding--right js-drawer-toggle"  id="tab1" aria-controls="tab1-section" aria-expanded="true">
+                        <span>Programme</span>
+                      </button>
+                    </div>
+                    <hr class="p-rule">
+                    <section class="p-accordion__panel is-dense" id="tab1-section" aria-hidden="false" aria-labelledby="tab1">
+                      {% include '/partners/partial/_partner-filters/_programme-filters.html' %}
+                    </section>
+                  </li>
+                  <li class="p-accordion__group" id="technology-section">
+                    <div role="heading" aria-level="2">
+                      <button type="button" class="p-accordion__tab u-no-padding--right js-drawer-toggle" id="tab1" aria-controls="tab1-section" aria-expanded="true">
+                        <span>Technology</span>
+                      </button>
+                    </div>
+                    <hr class="p-rule">
+                    <section class="p-accordion__panel is-dense" id="tab1-section" aria-hidden="false" aria-labelledby="tab1">
+                      {% include '/partners/partial/_partner-filters/_technology-filters.html' %}
+                    </section>
+                  </li>
+                  <li class="p-accordion__group" id="services-section">
+                    <div role="heading" aria-level="2">
+                      <button type="button" class="p-accordion__tab u-no-padding--right js-drawer-toggle" id="tab1" aria-controls="tab1-section" aria-expanded="true">
+                        <span>Services offered</span>
+                      </button>
+                    </div>
+                    <hr class="p-rule">
+                    <section class="p-accordion__panel is-dense" id="tab1-section" aria-hidden="false" aria-labelledby="tab1">
+                      {% include '/partners/partial/_partner-filters/_programme-filters.html' %}
+                    </section>
+                  </li>
+                </ul>
+              </aside>
+            </div>
+          </nav>
         </div>
       </div>
     </div>
+    <!-- End mobile side nav -->
   </section>
-  <section class="p-strip--light is-shallow js-search-form  u-hide" style="margin-top: -1px;">
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width">
     <div class="row">
```

Comment:
```suggestion
    <div class="row">
        <hr class="p-rule">
```

Reply:
is this for the hrs? It should preferably be u-fixed-width>hr, so it doesn't create a grid context

Reply:
The row is already there, why not use it?